pandas/tseries/offsets.py

```diff
@@ -1090,12 +1090,17 @@ def apply(self, other):
 
 
 class CustomBusinessMonthEnd(_CustomBusinessMonth):
-    __doc__ = _CustomBusinessMonth.__doc__.replace('[BEGIN/END]', 'end')
+    # TODO: Replace condition with Subsitution after dropping Py27 support
```

Comment:
ideally we could standarize these todos so can find later, maybe TODO(py27), I think we are doing similar to EA t